spec/controllers/api/v1/groups_controller_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Api::V1::GroupsController, :type => :controller do
+
+end
```

Comment:
yeh, I just moved them to the correct file. When I get a chance I can add some controller specs seeing as my changes may affect the results of the endpoints.